### PR TITLE
new uk index

### PR DESCRIPTION
**Pull request by bdvorianov**

### CODE REVIEW

File: uk/index.md [hunk 1]
```diff
@@ -1,29 +1,37 @@
 ---
-title: Рішення для Type Challenges
+title: Type Challenges Solutions
 description: >-
-  Проект має на меті допомогти вам краще зрозуміти, як працює система типів, як
-  писати свої типи, або просто гарно провести вечір за задачками.
-keywords: 'type, challenges, typescript, javascript, рішення'
+  Проект має на меті допомогти вам краще зрозуміти, як працює система типів,
+  як писати свої типи, або просто гарно провести вечір за задачками.
+keywords: 'type, challenges, solutions, typescript, javascript'
```
Comment (bdvorianov) [inline uk/index.md:6]:
```suggestion
keywords: 'type, challenges, розв'язок, typescript, javascript'
```

File: uk/index.md [hunk 2]
```diff
@@ -1,29 +1,37 @@
 ---
-title: Рішення для Type Challenges
+title: Type Challenges Solutions
```
Comment (bdvorianov) [inline uk/index.md:2]:
```suggestion
title: Розв'язки Type Challenges
```

**Review by ghaiklor — DISMISSED** [review]
LGTM

@shalamai what do you think?

**Comment by ghaiklor** [comment]
@bdvorianov you've broken a front matter section by moving "як" 🙂 
I'll push an update.

**Review by shalamai — APPROVED** [review]
👍

**Review by ghaiklor — APPROVED** [review]

**Pull request merged**